src/safeposix/syscalls/net_calls.rs

```diff
@@ -1765,22 +1765,102 @@ impl Cage {
         return 0;
     }
 
+    /// ### Description
+    /// 
+    /// `accept_syscall` accepts a connection on a socket 
+    /// 
+    /// ### Arguments
+    /// 
+    /// it accepts two parameters: 
+    /// * `fd` - the file descriptor that refers to the listening socket
+    /// * `addr` - the address of the peer socket (i.e. the client that is connecting)
+    ///            // ** Do we deal with the case in which addr may be NULL ?? ** //
+    /// 
+    /// ### Returns
+    /// 
+    /// for a successful call, the return value will be a file descriptor for the
+    /// accepted socket (a nonnegative integer). On error, -errno is
```

Comment:
better phrasing for the error part? "On error, a negative error number is returned, with the `errorno` set to represent the corresponding error"